activerecord/test/cases/associations/belongs_to_associations_test.rb

```diff
@@ -23,6 +23,9 @@
 require 'models/ship'
 require 'models/treasure'
 require 'models/parrot'
+require 'models/moon'
+require 'models/planet'
+require 'models/star'
```

Comment:
Is it possible to use existing models - for example `Star => Author`, `Planet => Post` and `Moon => Comment` ? If we keep adding models then it soon gets unmanageable.